chapters/fa/chapter2/2.mdx

```diff
@@ -260,21 +260,21 @@ print(outputs.last_hidden_state.shape)
 </div>
 
 
-توجه کنید که خروجی‌های ترنسفورمرهای هاگینگ‌فِیس، مانند `namedtuple`‌ها یا دیکشنری‌ها هستند. شما می‌توانید به هر عضو، با استفاده از نامش(مانند آنچه ما انجام دادیم) یا با کلیدش(`outputs["last_hidden_state"]`) یا حتی اگر دقیقاً از مکان آن اطلاع دارید با شماره‌اش(`outputs[0]`) دسترسی پیدا کنید.
+توجه کنید که خروجی‌های ترنسفورمرهای هاگینگ‌فِیس، رفتاری شبیه `namedtuple`‌ یا دیکشنری‌ دارند. شما می‌توانید به هر عضو، با استفاده از نامش(مانند آنچه ما انجام دادیم) یا با کلیدش(`outputs["last_hidden_state"]`) یا حتی اگر دقیقاً از مکان آن اطلاع دارید با اندیس‌اش(`outputs[0]`) دسترسی پیدا کنید.
 
-### سر مدل: درک اعداد درون مدل
+### سَر مدل: درک اعداد درون مدل
 
-قسمت سر، بردارهای چندین بعدی وضعیت پنهان را به عنوان ورودی می‌پذیرد و آن‌ها را به ابعادی در فضایی دیگر برای بخش بعدی پروجکت می‌کند. سرها معمولا از یک یا چند لایه خطی تشکیل شده‌اند.
+قسمت سَر، بردارهای چندین بعدی وضعیت پنهان را به عنوان ورودی می‌پذیرد و آنها را به بُعدی دیگر برای بخش بعدی پروجکت می‌کند. سَرها معمولا از یک یا چند لایه خطی تشکیل شده‌اند.
```

Comment:
"بردارهای چندین بعدی وضعیت پنهان" -> "بردارهای بُعدبالای وضعیت پنهان"

Just another proposal to (maybe) clarify the main message of the first line (totally up to you):
قسمت سَر، بردارهای بُعدبالای وضعیت پنهان را به عنوان ورودی می‌پذیرد و همچون یک مبدل، آنها را به بُعدی دیگر می‌برد.

Reply:
Fixed the first issue.

The second, I like your suggestion of می‌برد very much for "project". The added clause I'm not sure about. Went with:
قسمت سَر، بردارهای بُعد بالای وضعیت پنهان را به عنوان ورودی می‌پذیرد و آنها را به بُعدی دیگر می‌برد

 بُعد بالا 
has full space in the middle, per G.

Reply:
> بُعد بالا has full space in the middle, per G.

Ok, I believe this one is a little tricky @jowharshamshiri. Considering a full space per G guideline, maybe we'd better do more here to prevent a misunderstanding regarding "بالا" (conveying the 'above' meaning), so that the reader wouldn't go with the following interpretation:
...بردارهای بُعد 'بالایِ وضعیت پنهان' را...

This is my suggestion:
...بردارهای «بُعد بالای» وضعیت پنهان را...

Reply:
Fair enough. what if we add more diacritics to بعد بالا in G?

بردارهای بُعدِ بالای وضعیت پنها

Reply:
> Fair enough. what if we add more diacritics to بعد بالا in G?
> 
> بردارهای بُعدِ بالای وضعیت پنها

@jowharshamshiri This would make it more complicated I guess. I think we should revert back the change you made in G.
The term "بُعدِ بالا" means more like "High Dimension" or "Higher Dimension", rather than "High Dimensional". That's actually why I was advocating for "بُعدبالا" without full space.

Reply:
Take a look at these links:
https://bazidone.com/product/4-volume-collection-hard-books-child-poetry/
https://www.neekpey.com/hardcover-books-for-kids/
http://af.kfpaperbag.com/info/the-real-distinction-between-hardcover-and-sof-49328607.html

بردارهای بُعد بالا
are like کتاب‌های جلد سخت right? There is no ambiguity there and none here. You don't see people write کتاب‌های جلدسخت bc those are two separate words but you might see them add the diacritic like کتاب‌های جلدِ سخت. I agree that the new diacritic makes it more complicated and am in favor of removing it. I didn't understand the brackets you proposed bc it serves no purpose. There is no ambiguity. So remove the new diacritic?

Reply:
Yes, you're right. But the ambiguity rises when we use the term along side other words, specially when prepositions and postpositions like "ی" are added. See the below example, both interpretations are possible:

- بردارهای بُعد بالای : high dimensional vectors (belonging to...)
- بردارهای بُعد بالای : dimensional vectors above

Anyways, as you mentioned, reverting back and removing the new diacritic would be the right thing to do here. 👍